File: Consulta de Representación del conocimiento/Reprecentacion del conocimiento/Reprecentacion del conocimiento Java/src/reprecentacion/del/conocimiento/Animal.java
package reprecentacion.del.conocimiento;

/**
 *
 * @author Daniel Aviles, DeltaTeam, DCCO-ESPE
 */
public class Animal {
    private String especie;
    private int edad;
    private String nombre;
    
    public Animal(String especie, int edad, String nombre) {
        this.especie = especie;
        this.edad = edad;
        this.nombre = nombre;
    }
    
    public void mostrarInformacion() {
        System.out.println("Especie: " + especie);
        System.out.println("Edad: " + edad + " años");
        System.out.println("Nombre: " + nombre);
    }
}

File: Consulta de Representación del conocimiento/Reprecentacion del conocimiento/RDC c++/RDC c++/Animal.cpp
#include "Animal.h"
Animal::Animal(std::string especie, int edad, std::string nombre) {
    this->especie = especie;
    this->edad = edad;
    this->nombre = nombre;
}

void Animal::mostrarInformacion() {
    std::cout << "Especie: " << especie << std::endl;
    std::cout << "Edad: " << edad << " años" << std::endl;
    std::cout << "Nombre: " << nombre << std::endl;
}